src/main/java/io/github/wimdeblauwe/errorhandlingspringbootstarter/mapper/ErrorCodeMapper.java

```diff
@@ -38,6 +34,24 @@ public String getErrorCode(Throwable exception) {
         }
     }
 
+    private String getErrorCode(Class<?> exceptionClass) {
```

Comment:
Private methods should be below public methods.

Reply:
Ok.  I was just keeping related code near but no problem changing that.